### PR TITLE
Create cdn.shymike.json

### PR DESCRIPTION
**Pull request by ImShyMike**

### CODE REVIEW

File: domains/cdn.shymike.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "ImShyMike",
+        "email": "",
+        "discord": "582648583635992622"
+    },
+    "record": {
+        "A": [
+            "69.197.135.203"
```
Comment (Stef-00012) [inline domains/cdn.shymike.json:9]: error 502 Bad Gateway

**Review by Stef-DP — DISMISSED** [review]

File: domains/cdn.shymike.json (hunk 1 above)
Comment (ImShyMike) [reply domains/cdn.shymike.json:9]: its from danbot thats why :-:

Comment (ImShyMike) [reply domains/cdn.shymike.json:9]: nvm i configured it wrong

**Review by iostpa — APPROVED** [review]
Uh so this is hosted in DBH and user wants to use it for cdn stuff, so I'll merge this PR

**Pull request merged**